pkg/tanka/errors.go

```diff
@@ -0,0 +1,22 @@
+package tanka
+
+import "fmt"
+
+// ErrNoEnv means that the given jsonnet has no Environment object
+// This must not be fatal, some operations work without
+type ErrNoEnv struct {
+	path string
+}
+
+func (e ErrNoEnv) Error() string {
+	return fmt.Sprintf("unable to find an Environment in '%s'", e.path)
+}
+
+// ErrMultipleEnvs means that the given jsonnet has multiple Environment objects
+type ErrMultipleEnvs struct {
```

Comment:
Could this error message include the names of the environments? Makes it far easier to debug then

Reply:
https://github.com/grafana/tanka/pull/403/commits/065a3e1c6606861f2e8bfc2e5a931cf75d796c1a